egg-herbie/src/lib.rs

```diff
@@ -131,6 +148,12 @@ pub unsafe extern "C" fn egraph_run_with_iter_limit(
         context.rules = rules;
 
         context.runner.egraph.analysis.constant_fold = is_constant_folding_enabled;
+        context.runner = if simple_scheduler {
+            context.runner.with_scheduler(SimpleScheduler)
+        } else {
+            context.runner.with_scheduler(BackoffScheduler::default())
+        };
```

Comment:
Why do we need to control the scheduler? Just in case, or do we use this?

Reply:
We use the simple scheduler to fully apply lowering rewrites